File: MED-CLOUD/med-data/src/main/java/nta/med/data/model/ihis/ocsa/OCS0108U00grdOCS0103ItemInfo.java
package nta.med.data.model.ihis.ocsa;

import java.sql.Timestamp;

public class OCS0108U00grdOCS0103ItemInfo {
	private String slipCode ;    
	private String hangmogCode ; 
	private String hangmogName  ;  
	private String sunabDanui;         
	private String sunabDanuiName;   
	private String subulDanui  ;      
	private String subulDanuiName ;  
	private Timestamp hangmogStartDate ;
	public OCS0108U00grdOCS0103ItemInfo(String slipCode, String hangmogCode,
			String hangmogName, String sunabDanui, String sunabDanuiName,
			String subulDanui, String subulDanuiName, Timestamp hangmogStartDate) {
		super();
		this.slipCode = slipCode;
		this.hangmogCode = hangmogCode;
		this.hangmogName = hangmogName;
		this.sunabDanui = sunabDanui;
		this.sunabDanuiName = sunabDanuiName;
		this.subulDanui = subulDanui;
		this.subulDanuiName = subulDanuiName;
		this.hangmogStartDate = hangmogStartDate;
	}
	public String getSlipCode() {
		return slipCode;
	}
	public void setSlipCode(String slipCode) {
		this.slipCode = slipCode;
	}
	public String getHangmogCode() {
		return hangmogCode;
	}
	public void setHangmogCode(String hangmogCode) {
		this.hangmogCode = hangmogCode;
	}
	public String getHangmogName() {
		return hangmogName;
	}
	public void setHangmogName(String hangmogName) {
		this.hangmogName = hangmogName;
	}
	public String getSunabDanui() {
		return sunabDanui;
	}
	public void setSunabDanui(String sunabDanui) {
		this.sunabDanui = sunabDanui;
	}
	public String getSunabDanuiName() {
		return sunabDanuiName;
	}
	public void setSunabDanuiName(String sunabDanuiName) {
		this.sunabDanuiName = sunabDanuiName;
	}
	public String getSubulDanui() {
		return subulDanui;
	}
	public void setSubulDanui(String subulDanui) {
		this.subulDanui = subulDanui;
	}
	public String getSubulDanuiName() {
		return subulDanuiName;
	}
	public void setSubulDanuiName(String subulDanuiName) {
		this.subulDanuiName = subulDanuiName;
	}
	public Timestamp getHangmogStartDate() {
		return hangmogStartDate;
	}
	public void setHangmogStartDate(Timestamp hangmogStartDate) {
		this.hangmogStartDate = hangmogStartDate;
	}
}

File: IHIS/IHIS.Framework/IHIS.CloudConnector/Contracts/Models/Ocsa/OCS0108U00grdOCS0103ItemInfo.cs
using System;

namespace IHIS.CloudConnector.Contracts.Models.Ocsa
{
    [Serializable]
	public class OCS0108U00grdOCS0103ItemInfo
	{
		private String _slipCode;
		private String _hangmogCode;
		private String _hangmogName;
		private String _sunabDanui;
		private String _sunabDanuiName;
		private String _subulDanui;
		private String _subulDanuiName;
		private String _hangmogStartDate;

		public String SlipCode
		{
			get { return this._slipCode; }
			set { this._slipCode = value; }
		}

		public String HangmogCode
		{
			get { return this._hangmogCode; }
			set { this._hangmogCode = value; }
		}

		public String HangmogName
		{
			get { return this._hangmogName; }
			set { this._hangmogName = value; }
		}

		public String SunabDanui
		{
			get { return this._sunabDanui; }
			set { this._sunabDanui = value; }
		}

		public String SunabDanuiName
		{
			get { return this._sunabDanuiName; }
			set { this._sunabDanuiName = value; }
		}

		public String SubulDanui
		{
			get { return this._subulDanui; }
			set { this._subulDanui = value; }
		}

		public String SubulDanuiName
		{
			get { return this._subulDanuiName; }
			set { this._subulDanuiName = value; }
		}

		public String HangmogStartDate
		{
			get { return this._hangmogStartDate; }
			set { this._hangmogStartDate = value; }
		}

		public OCS0108U00grdOCS0103ItemInfo() { }

		public OCS0108U00grdOCS0103ItemInfo(String slipCode, String hangmogCode, String hangmogName, String sunabDanui, String sunabDanuiName, String subulDanui, String subulDanuiName, String hangmogStartDate)
		{
			this._slipCode = slipCode;
			this._hangmogCode = hangmogCode;
			this._hangmogName = hangmogName;
			this._sunabDanui = sunabDanui;
			this._sunabDanuiName = sunabDanuiName;
			this._subulDanui = subulDanui;
			this._subulDanuiName = subulDanuiName;
			this._hangmogStartDate = hangmogStartDate;
		}

	}
}